pkg/helm/helm.go

```diff
@@ -52,8 +51,14 @@ func GetHelmURL(arch, os, version string) string {
 	return fmt.Sprintf("https://get.helm.sh/helm-%s-%s-%s.tar.gz", version, osSuffix, archSuffix)
 }
 
-func DownloadHelm(userPath, clientArch, clientOS, subdir string) error {
+func DownloadHelm(userPath, clientArch, clientOS, subdir string, helm3 bool) error {
+
 	useHelmVersion := helmVersion
+	if helm3 {
+		useHelmVersion = helm3Version
+	}
+
+	// This is an override users can set, its documented in the README
 	if val, ok := os.LookupEnv("HELM_VERSION"); ok && len(val) > 0 {
```

Comment:
I'm still seeing HELM_VERSION being used here? Won't it have the same issues as when we set it in each app?

Reply:
This is to keep the override option thats documented in the README, the apps no longer set the "HELM_VERSION"